x-pack/test/ui_capabilities/common/services/ui_capabilities.ts

```diff
@@ -42,21 +43,23 @@ export class UICapabilitiesService {
 
   public async get({
     credentials,
-    navLinks,
     spaceId,
   }: {
     credentials?: BasicCredentials;
-    navLinks?: Record<string, boolean>;
     spaceId?: string;
   }): Promise<GetUICapabilitiesResult> {
     const spaceUrlPrefix = spaceId ? `/s/${spaceId}` : '';
-    this.log.debug(`requesting ${spaceUrlPrefix}/api/capabilities to get the uiCapabilities`);
-    const requestOptions = credentials ? { auth: credentials } : {};
-    const response = await this.axios.post(
-      `${spaceUrlPrefix}/api/capabilities`,
-      { capabilities: { navLinks } },
-      requestOptions
-    );
+    this.log.debug(`requesting ${spaceUrlPrefix}/app/kibana to parse the uiCapabilities`);
+    const requestHeaders = credentials
+      ? {
+          Authorization: `Basic ${Buffer.from(
+            `${credentials.username}:${credentials.password}`
+          ).toString('base64')}`,
+        }
+      : {};
+    const response = await this.axios.get(`${spaceUrlPrefix}/app/kibana`, {
```

Comment:
Before this was using `post`, but is now using `get`. Is that a problem for this route?

Reply:
No it's now a different route that actually pulls the uiCapabilities from the HTML's injectedVars. I mostly just reverted the test changes from my previous PR to make this work.